### PR TITLE
Introduce process argument support

### PR DESCRIPTION
**Pull request by yurii-glia**
Supporting process argument allows passing configuration url before installing the application and running AT. These changes should speed up and simplify the testing process.

### CODE REVIEW

**Review by EgorovEI — APPROVED** [review]
That awesome! Now we are able to add AT for screen sharing 🔥

**Review by gersonnoboa — APPROVED** [review]

**Review by igorkravchenko — APPROVED** [review]

**Review by rasmustautsglia — APPROVED** [review]

File: GliaWidgets/Sources/Component/Bubble/BubbleView.swift [hunk 1]
```diff
@@ -158,7 +154,7 @@ class BubbleView: UIView {
         accessibilityIdentifier = isVisitorOnHold ? "on_hold_bubble_view" : "bubble_view"
     }
 
-    @objc private func tapped() {
+    @objc func tapped() {
```
[the remark this answers is not in the document]
Comment (yurii-glia) [reply GliaWidgets/Sources/Component/Bubble/BubbleView.swift]: fixed

File: TestingApp/AppDelegate.swift [hunk 2]
```diff
@@ -46,4 +36,43 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     ) {
         debugPrint("💥 Failed to register for remote notifications. Error: \(error.localizedDescription)")
     }
+
+    func handleProcessInfo() {
```
[the remark this answers is not in the document]
Comment (yurii-glia) [reply TestingApp/AppDelegate.swift]: fixed

File: GliaWidgets/Sources/ViewController/GliaViewController.swift [hunk 3]
```diff
@@ -115,25 +116,15 @@ class GliaViewController: UIViewController {
...
-            } else {
-                return BubbleWindow(
-                    bubbleView: bubbleView,
-                    environment: .init(
-                        uiScreen: environment.uiScreen,
-                        uiApplication: environment.uiApplication
-                    )
-                )
-            }
+        if let windowScene = windowScene() {
```
[the remark this answers is not in the document]
Comment (yurii-glia) [reply GliaWidgets/Sources/ViewController/GliaViewController.swift:119]: 🎉

**Pull request merged**